go/libkb/util_windows.go

```diff
@@ -183,6 +238,10 @@ func RemoteSettingsRepairman(g *GlobalContext) error {
 	oldConfig := filepath.Join(oldHome, configName)
 	if oldExists, _ := FileExists(oldConfig); oldExists {
 		if currentExists, _ := FileExists(currentConfig); !currentExists {
+			g.Log.Info("RemoteSettingsRepairman moving from %s to %s", oldHome, currentHome)
```

Comment:
I disagree, I think any efforts to automatically repair people who are halfway moved will end in disaster. They may have reprovisioned that device, etc.